### DISCUSSION TITLE
Proper way to update the Windows App

### DISCUSSION THREAD

**Discussion by eliasbats** [xournalpp/xournalpp#5669] [General] [answered]
What is the proper way to update the windows app? I just download and run the new installation package? Will my configuration and settings be preserved? Is there an update procedure in the documentation? Because I haven't been able to find one. If not, maybe there should be one.

**Comment by eliasbats** [comment]
Just wanted to add that from a humble user's perspective the application rocks... thanks to all contributors... 🙏

**Comment by rolandlo** [comment, marked as answer]
Simply installing the new version over the old one should work, as the installer script first removes the old version (and even handles legacy versions that were structured slightly differently).

The configurations and settings will be preserved as they are stored in the config folder, see https://xournalpp.github.io/guide/file-locations/, and untouched by installation/deinstallation.
The installation/uninstallation process on Windows is handled in the [nsis installation script](https://github.com/xournalpp/xournalpp/blob/master/windows-setup/xournalpp.nsi), so if you want to learn about the details take a look there.

**Reply by JaiWWW** [reply]
I second that it would be nice if the readme (or perhaps a separate readme) could just clarify the expected/recommended update procedure for each platform! Luckily I found this post though so I didn't have to ask or trial and error it

**Comment by eliasbats** [comment]
Thank you!
